systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -407,10 +410,10 @@ def fw_vpcrouter(self):
                             ])
 
         if self.get_type() in ["public"]:
-            self.fw.append(["nat", "front",
-                            "-A POSTROUTING -o %s -j SNAT --to-source %s" %
-                           (self.dev, self.address['public_ip'])
-                            ])
+            # self.fw.append(["nat", "front",
```

Comment:
+1

Can I do all that in the new PR along with the fixes?

I mean... I will have 1 commit removing all the commented out lines. Is it okay?

:)